### PR TITLE
NXP-26185 - Documentation on the Search form element

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

**Comment by guirenard** [comment]
Please use a NXP ticket

**Comment by gcarlin** [comment]
Commit message updated

**Comment by guirenard** [comment]
Thanks. I am gonna be picky but we usually prefix with `NXP-26185:` :)

File: elements/search/nuxeo-search-form.html [hunk 1]
```diff
@@ -390,27 +392,30 @@ <h2>[[i18n('searchForm.delete.heading')]]</h2>
...
-         * The page size passed on to `provider`.
+         * The page size passed on to `provider` (number of results per page).
          **/
         pageSize: {
           type: Number,
           value: 40
         },
         /**
          * The parameters passed on to `provider`.
+         * Useful to do a mapping between the query parameters and the variables
```
Comment (guirenard) [inline elements/search/nuxeo-search-form.html]: Please try to keep sentence consistent and put a period at the end of everyone

File: elements/search/nuxeo-search-form.html [hunk 2]
```diff
@@ -497,6 +508,7 @@ <h2>[[i18n('searchForm.delete.heading')]]</h2>
         },
         /**
          * The name of the search layout.
+         * Usually the page provider name
```
Comment (guirenard) [inline elements/search/nuxeo-search-form.html]: same

File: elements/search/nuxeo-search-form.html (hunk 1 above)
Comment (gcarlin) [reply elements/search/nuxeo-search-form.html]: fixed

File: elements/search/nuxeo-search-form.html (hunk 2 above)
Comment (gcarlin) [reply elements/search/nuxeo-search-form.html]: fixed

**Comment by guirenard** [comment]
:+1:

**Comment by nmpcunha** [comment]
👍

**Pull request merged**